config/routes.rb

```diff
@@ -63,4 +63,11 @@
       get 'history', to: 'work_histories#show'
     end
   end
+
+  namespace :api do
+    resources :ingest, only: [:create], controller: 'v1/ingest'
```

Comment:
I would propose removing this route, and only supporting the versioned route

Reply:
@rschenk Yeah, I was thinking having a default route for the user would be easier, but it's better to be explicit.